<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>911</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <item>
      <layout class="QVBoxLayout" name="graphLayers">
       <item>
        <widget class="QListWidget" name="graphLayerList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="graphLayerActions">
         <item>
          <widget class="QPushButton" name="addGraphLayer">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteGraphLayer">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOuter"/>
   <addaction name="actionInner"/>
   <addaction name="actionPalette"/>
   <addaction name="actionErase"/>
   <addaction name="separator"/>
   <addaction name="actionMove"/>
   <addaction name="actionRotate"/>
   <addaction name="actionZoom"/>
   <addaction name="actionVflip"/>
   <addaction name="actionHflip"/>
   <addaction name="separator"/>
   <addaction name="actionClip"/>
  </widget>
  <action name="actionMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbaricon.qrc">
     <normaloff>:/new/icon/toolbar/icon/toolbar/move.png</normaloff>:/new/icon/toolbar/icon/toolbar/move.png</iconset>
   </property>
   <property name="text">
    <string>move</string>
   </property>
   <property name="toolTip">
    <string>Drag to move.</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+M</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbaricon.qrc">
     <normaloff>:/new/icon/toolbar/icon/toolbar/zoom.png</normaloff>:/new/icon/toolbar/icon/toolbar/zoom.png</iconset>
   </property>
   <property name="text">
    <string>zoom</string>
   </property>
   <property name="toolTip">
    <string>Scroll to zoom.</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+Z</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbaricon.qrc">
     <normaloff>:/new/icon/toolbar/icon/toolbar/rotate.png</normaloff>:/new/icon/toolbar/icon/toolbar/rotate.png</iconset>
   </property>
   <property name="text">
    <string>rotate</string>
   </property>
   <property name="toolTip">
    <string>Drag to rotate.</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+R</string>
   </property>
  </action>
  <action name="actionVflip">
   <property name="icon">
    <iconset resource="toolbaricon.qrc">
     <normaloff>:/new/icon/toolbar/icon/toolbar/vflip.png</normaloff>:/new/icon/toolbar/icon/toolbar/vflip.png</iconset>
   </property>
   <property name="text">
    <string>vflip</string>
   </property>
   <property name="toolTip">
    <string>Click to flip vertically.</string>
   </property>
   <property name="shortcut">
    <string>Meta+F</string>
   </property>
  </action>
  <action name="actionHflip">
   <property name="icon">
    <iconset resource="toolbaricon.qrc">
     <normaloff>:/new/icon/toolbar/icon/toolbar/hflip.png</normaloff>:/new/icon/toolbar/icon/toolbar/hflip.png</iconset>
   </property>
   <property name="text">
    <string>hflip</string>
   </property>
   <property name="toolTip">
    <string>Click to flip horizontally.</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+F</string>
   </property>
  </action>
  <action name="actionClip">
   <property name="icon">
    <iconset resource="toolbaricon.qrc">
     <normaloff>:/new/icon/toolbar/icon/toolbar/clip.png</normaloff>:/new/icon/toolbar/icon/toolbar/clip.png</iconset>
   </property>
   <property name="text">
    <string>clip</string>
   </property>
   <property name="toolTip">
    <string>Choose two polygons to clip.</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+C</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbaricon.qrc">
     <normaloff>:/new/icon/toolbar/icon/toolbar/polygon.png</normaloff>:/new/icon/toolbar/icon/toolbar/polygon.png</iconset>
   </property>
   <property name="text">
    <string>polygon</string>
   </property>
   <property name="toolTip">
    <string>Click to draw polygon.</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+P</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="icon">
    <iconset resource="toolbaricon.qrc">
     <normaloff>:/new/icon/toolbar/icon/toolbar/erase.png</normaloff>:/new/icon/toolbar/icon/toolbar/erase.png</iconset>
   </property>
   <property name="text">
    <string>erase</string>
   </property>
   <property name="toolTip">
    <string>Click to erase</string>
   </property>
  </action>
  <action name="actionOuter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbaricon.qrc">
     <normaloff>:/new/icon/toolbar/icon/toolbar/polygon_outer.png</normaloff>:/new/icon/toolbar/icon/toolbar/polygon_outer.png</iconset>
   </property>
   <property name="text">
    <string>outer</string>
   </property>
   <property name="toolTip">
    <string>Click to draw outer polygon.</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+O</string>
   </property>
  </action>
  <action name="actionInner">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbaricon.qrc">
     <normaloff>:/new/icon/toolbar/icon/toolbar/polygon_inner.png</normaloff>:/new/icon/toolbar/icon/toolbar/polygon_inner.png</iconset>
   </property>
   <property name="text">
    <string>inner</string>
   </property>
   <property name="toolTip">
    <string>Click to draw inner polygon.</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+I</string>
   </property>
  </action>
  <action name="actionPalette">
   <property name="icon">
    <iconset resource="toolbaricon.qrc">
     <normaloff>:/new/icon/toolbar/icon/toolbar/palette.png</normaloff>:/new/icon/toolbar/icon/toolbar/palette.png</iconset>
   </property>
   <property name="text">
    <string>palette</string>
   </property>
   <property name="toolTip">
    <string>Click to change color.</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="toolbaricon.qrc"/>
 </resources>
 <connections/>
</ui>
